spec/controllers/users/reactivate_profile_controller_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe Users::ReactivateProfileController do
+  include Features::LocalizationHelper
+
+  describe '#create' do
+    subject(:action) do
+      post :create,
+           reactivate_profile_form: { password: 'password', recovery_code: 'recovery_code' }
```

Comment:
this is super minor, but I would usually format this like:
```
post(
  :create,
  reactivate_profile_form: { password: 'password', recovery_code: 'recovery_code' }
)
```

...for easier readability of args (especially when there are more than 2, I find it really helps to see how many args there are). not sure if that is consistent with this rest of this proj, though